app/assets/javascripts/discourse/tests/acceptance/keyboard-shortcuts-test.js

```diff
@@ -85,8 +84,10 @@ acceptance("Keyboard Shortcuts - Authenticated Users", function (needs) {
   });
 
   test("dismiss unread from top and bottom button", async function (assert) {
-    // visit root first so topic list starts fresh
+    // need to scroll to top so the viewport shows the top of the page
+    // and top dismiss button
     await visit("/");
+    await (document.getElementById("ember-testing-container").scrollTop = 0);
```

Comment:
```suggestion
    document.getElementById("ember-testing-container").scrollTop = 0;
```

There's no Promise to await 🙂